test/run.ts

```diff
@@ -164,41 +169,53 @@ function generateOutputBlock(currentOutput: any, ast: any, printed: any, options
 
   let diagnosticString = '';
 
-  if (printed !== '✖ Soon to be open sourced') {
```

Comment:
Danger to remove this guard unless you have a printer code ;) Things will break if you can't print